v03_pipeline/lib/misc/io.py

```diff
@@ -71,6 +70,8 @@ def import_gcnv_bed_file(callset_path: str) -> hl.MatrixTable:
             hl.agg.collect_as_set(parse_gcnv_genes(mt.genes_LOF_Ensemble_ID)),
         ),
     )
+    mt = mt.unfilter_entries()
```

Comment:
This is the actual one line fix.

Reply:
yeah I hit this one too once, its a doozy